packages/repo-mock/src/__tests__/index.test.ts

```diff
@@ -62,6 +62,12 @@ describe('mockRepo', () => {
     await expect(fetch(`${baseURL}/metadata/2.root.json`)).rejects.toThrow(
       /404/
     );
+
+    // No mock should be set-up for the 1.root.json file as this should never be
+    // fetched in a normal TUF flow.
+    await expect(fetch(`${baseURL}/metadata/1.root.json`)).rejects.toThrow(
```

Comment:
i am little bit confused. it does make sense then we don't need to fetch 1.root.json . however, i thought the code did setyp 1.root.json? 
https://github.com/theupdateframework/tuf-js/blob/22faf6805cc5923fc166a1d8b14400d685a3d9d0/packages/repo-mock/src/handler.ts#L18-L21

Reply:
The path to "1.root.json" **is** included in the set of handlers which are returned from the `tufHandlers` function . . . however, when setting up the nock-based mocks, I'm specifically excluding it:

https://github.com/theupdateframework/tuf-js/blob/22faf6805cc5923fc166a1d8b14400d685a3d9d0/packages/repo-mock/src/index.ts#L27-L35

In the unit test mocking scenario there should never be a request for "1.root.json" cause this will already be seeded in the TUF cache. However, I also want to use this collection of handlers to create an actual Sigstore "server" for more integration-style tests. In this scenario I'd like to be able to retrieve the "1.root.json" from the HTTP endpoint in the same way you can from a real TUF repo.

The primary reason for this refactor is that I want to be able to re-use this functionality along with my Fulcio and Rekor mocks to create a complete Sigstore stack running on a single endpoint.

Reply:
gotcha!